### DISCUSSION TITLE
keep the sides also while comparing files

### DISCUSSION THREAD

**Discussion by muellerto** [doublecmd/doublecmd#1874] [Suggestions]
When you compare a file on the left side with a file on the right side it would be a great help when the file compare window could also use these sides, left file on the left, right file on the right.

Currently it depends somehow on the selection order. The first selected file appears always on the left side, the second selected file on the right side, which can be different from the panes. So you must always have a concentrated look which file is which or it happens that you modify the wrong file.

**Comment by alexx2000** [comment]
[Position of frame panel after the comparison](https://doublecmd.github.io/doc/en/configuration.html#ConfigToolsDiffer)

**Comment by muellerto** [comment]
Argh.
